index.html

```diff
@@ -12,40 +12,289 @@
 
 <body>
     <!-- Header -->
-    <div class="header">
-        <a href="#" class="logo">My Logo</a>
-        <a href="#" class="icon" >
-            <i class="fa-solid fa-bars"></i>
-        </a>
+
+        <div class="header">
+            <a href="#" class="logo">My Logo</a>
+            <a href="#" class="icon" >
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </div>
+        <!-- end of header -->
+        <!-- landing page -->
+    <div class="">
+        <section class="hero">
+            <h1> Hey There.<br>I'm Jenadius </h1>
+            <h5 >I'm a Software Developer.</h5>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+            see and have a project you need coded,
+            don't hestiate to contact me.</p>
+            <ul class="link-container">
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-github"></i></a></li>
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-reddit-alien"></i></a></li>
+            </ul>
+        </section>
+        <!-- end of landing page -->
+
+    <!-- main -->
+        <section class="main">
+            <div id="recent-work" class="recent-work">
+                <h1 class="title">My recent works</h1>
+                <hr class="recent-work-line">
+            </div>
+            <div class="image-placeholder">
+                <img class="imge" src="assets/img/imageplaceholder.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="teck-stacks">
+                    <ul class="ul-teck-stacks">
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">css </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">html </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Bootstrap </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Ruby </button></li>
+                    </ul>
+                </div>
+                <div class="btn-see-project">
+                    <button type="button" class="">See Project </button>
+                </div>
```

Comment:
Figma design           |  Your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/153716370-3160ebeb-0f0c-4208-90dd-3ca89a67b4fa.png) |  ![hisworks](https://user-images.githubusercontent.com/69638013/153716423-52ff2822-03d5-4252-af9d-747d6fdf07bb.PNG)

- [ ] Kindly use the appropriate font colors, and sizes for these content check  screenshot above ⬆️ 
- [ ] Kindly add hover effect to button